### ISSUE TITLE
Forking survey-creator

### ISSUE THREAD

**Issue by frankinedinburgh** [surveyjs/survey-creator#2075]
### How to fork this project ?
When I fork this project, run npm install and npm run bootstrap I get a missing dependency error. I have survey-library and survey-creator cloned in the same directory. Also 
packages/survey-creator-knockout/package.json
tries to import
`./../../survey-library/build/survey-knockout-ui` from survey-library but this directory does not exist in the survey-library repo

Can instructions be added to the README detailing how to setup this project locally ?

### What is the current behavior?

```lerna ERR! npm install exited 1 in 'survey-creator-core```

### What is the expected behavior?
I fork the repo. I run npm install, npm run bootstrap and all dependencies are installed in the packages directory

### How would you reproduce the current behavior (if this is a bug)?
Fork this repo, clone survey-library in the same directory . Run npm instal and npm run bootstrap

### Specify your

* browser:
Windows_NT 10
node v14.17.1
npm  v7.20.5

**Comment by tsv2013** [comment]
Yes, the instruction is outdated a bit. It raleted to the SurveyJS Creator V1 which is resides in the `packages/survey-creator` subfolder. You can built it via "npm i" and then "npm run build_prod" commands.

The other projects from this repo are related to Survey Creator V2 and right now in active development and build instructions will be published after release.

Note that SurveyJS Creator is a commercial product and you need to buy a license to use it in your applications.

**Comment by AndOne1974** [comment]
I have the same problem under linux.
